crates/net/discv4/src/proto.rs

```diff
@@ -209,6 +211,76 @@ pub struct Neighbours {
     pub expire: u64,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct EnrWrapper<K: EnrKey>(Enr<K>);
```

Comment:
let's document why we need to wrap this.
essentially: can't implement a foreign trait on a foreign type.